server.js

```diff
@@ -38,6 +41,14 @@ mongoose
 
 // Routes
 app.use('/users', usersRouter);
+app.use('/signup', signupRouter);
+app.use('/login', loginRouter);
```

Comment:
Hi good work. We could put this too in the same route and call the function `authRouter`, so we would have just one call to use for both:

```
app.use('/', authRouter)
```

and then you use `/login` and `singup` in the `authRouter` js. Sorry about all this trouble. I think if it is organized will save us some trouble later.

Reply:
done